packages/runner/bin/runner.js

```diff
@@ -4,15 +4,19 @@ import chalk from 'chalk';
 import open from 'open';
 import { exec } from 'child_process';
 import getPort from 'get-port';
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
 import { stringIsAValidUrl, clearConsole } from '../src/util.js';
 import setupFrontends from '../src/setupFrontends.js';
 import setupServer from '../src/setupServer.js';
 
+const parentFolder = dirname(fileURLToPath(import.meta.url));
+const runnerFrontendPath = join(parentFolder, '..', 'frontend');
+
 // TODO: MAIN-83 setup dev environment option for a local dummy user frontend and backend
 (async () => {
   program
     .addOption(new Option('-t, --tbg-frontend-url <tbgFrontendUrl>').hideHelp())
-    .addOption(new Option('-e, --empty-backend').hideHelp())
```

Comment:
We no longer need to handle this option, which simplifies our runner code, because we pulled out the test app into a separate package! Instead, we will run the dev script from test-app package to test the runner.